frontend/src/components/admin-page.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+import {
+  Nav,
+  NavList,
+  Page,
+  PageSidebar,
+  PageSidebarBody
+} from '@patternfly/react-core';
+
+import { Link, Outlet } from 'react-router-dom';
+import ElementWrapper from './elementWrapper';
+
+import { IbutsuHeader } from './ibutsu-header';
+import PropTypes from 'prop-types';
+
+
+
+const AdminPage = (props) => {
+    // TODO useEffect instead of eventEmitter prop
+    // TODO notifications on admin page with state and AlertGroup
+    const navigation = (
+      // TODO what is onNavSelect doing here ... I just carried this from a class ref
+      <PageSidebar theme="dark" >
+        <PageSidebarBody>
+          <Nav onSelect={React.Component.onNavSelect} theme="dark" aria-label="Nav">
```

Comment:
It seems that any of the admin or profile pages do not react to changes in dark mode, though changing it there does carry the changes over when switching to a page that does react to it

Reply:
does the behavior differ from prod ibutsu right now?

Reply:
Just checked - I did in fact break this here, will take a look at how to get it working better.

Reply:
https://www.patternfly.org/developer-resources/dark-theme-handbook#best-practices

Going to go over the entire app's handling of dark mode following these practices.

Reply:
Pushed a commit to address the theme handling. It might still be a little buggy though with re-renders and syncing to browser default on first render.